Changelog.md

```diff
@@ -19,6 +19,9 @@ Nothing here yet.
 - Remove unused `JalaliMixin`
 - Replace `pkg_resources` with `importlib_metadata` to fetch the version number in `__init__.py` (#657)
 - Added new badges (pypi, conda, license) and installation instructions (pip, conda) to readme file @sugatoray (#673).
+- Replace `pyCalverter` with `convertdate` (#536)
+- Remove unused `JalaliMixin`
+- Fixed United Kingdom's 2022 holidays ; Spring Bank Holiday has been moved to the 3rd of June and Queen's Platinum Jubilee added to 2nd of June.
```

Comment:
Sorry, but this changelog entry should be in the "master (unreleased)" section at the top of the file.